### PR TITLE
Add api endpoint to create a new subscription

### PR DESCRIPTION
**Pull request by kdeal**
This is essentially the same logic that we have in the
scripts/subscriptions tool, but as an api endpoint.

### CODE REVIEW

File: api/yelp_beans/routes/api/v1/subscriptions.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+@enum.unique
+class Weekday(enum.Enum):
+    monday = "monday"
+    tuesday = "tuesday"
+    wednesday = "wednesday"
+    thursday = "thursday"
+    friday = "friday"
+    saturday = "saturday"
+    sunday = "sunday"
```
[the remark this answers is not in the document]
Comment (kdeal) [reply api/yelp_beans/routes/api/v1/subscriptions.py]: Fixed.

**Comment by ny2ko** [comment]
Feel free to ignore this suggestion if this works better for Yelp. What I've done for the beans app at Twitch is just turn on Flask-Admin then I do all these changes using that UI. It does require a good understanding of the models though

**Comment by kdeal** [comment]
This is based off of this section of the subscriptios script. https://github.com/Yelp/beans/blob/master/api/scripts/subscriptions.py#L63

**Comment by kdeal** [comment]
> Feel free to ignore this suggestion if this works better for Yelp. What I've done for the beans app at Twitch is just turn on Flask-Admin then I do all these changes using that UI. It does require a good understanding of the models though

That would be a good quick solution. I think we are hoping to do something that doesn't require a good understand of models. 

ny2ko++

**Review by ymilki — APPROVED** [review]

**Pull request merged**